.gitattributes

```diff
@@ -0,0 +1,5 @@
+* text=auto eol=lf
+*.py text eol=lf
+*.toml text eol=lf
+*.md text eol=lf
+*.lock binary
```

Comment:
Remove this line please. I need to be able to see lockfile changes in PRs (GitHub collapses them by default anyway so they don't add noise).

In fact, since we don't have binary files in the repo at all (and I don't intend to add any), the whole `.gitattributes` can probably be:

```
* text eol=lf
```

I don't trust auto-detection and if something is strange I want to see it immediately rather than Git deciding I don't need to.